CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Incorrect line numbers when printing errors in TypeScript extensions
 - Absolute paths submitted when analyzing manifest files
 - Ecosystem extensions not pre-checking `remove`/`uninstall` operations
+- Disabled selfmanage commands in completion for Homebrew users
```

Comment:
"selfmanage commands" is internal terminology, so imo not ideal for a changelog entry.